src/Compilers/Core/MSBuildTaskTests/TestUtilities/EventArgsFormatting.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Is there a reason the head line is split?

Reply:
Deleted.